### DISCUSSION TITLE
Useful links

### DISCUSSION THREAD

**Discussion by ghost** [beerisgood/Windows11_Hardening#3] [General]
Hardening Microsoft Office:
https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-microsoft-365-office-2021-office-2019-and-office-2016

Attack surface reduction rules reference:
https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference?view=o365-worldwide

**Comment by beerisgood** [comment]
ASR rules (overview) is already included in "Hardening".

For Office part, the most secure solution is using Microsoft Online Office: https://www.office.com

**Reply by ghost** [reply]
Thank you for taking the time to reply and for your guides.

**Reply by beerisgood** [reply]
You’re welcome!
